games-util/gamemode/gamemode-1.2.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit linux-info meson multilib-minimal ninja-utils user
+
+DESCRIPTION="Optimise Linux system performance on demand"
+HOMEPAGE="https://github.com/FeralInteractive/gamemode"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/FeralInteractive/gamemode.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/FeralInteractive/gamemode/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="amd64 x86"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+
+RDEPEND="
+	>=sys-apps/systemd-236[${MULTILIB_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+# see "https://github.com/FeralInteractive/gamemode/blob/master/README.md"
+CONFIG_CHECK="~SCHED_MUQSS ~SCHED_PDS"
```

Comment:
Hmm, to be honest I don't think this is a good idea. Given that you can't expect more than one of them to be present, you'd be always outputting a warning, even if one of them is present.

Either use `linux_chkconfig*` functions directly to check for presence of symbols, and manually print warnings based on the appropriate `if`s, or just go for unconditional message in `pkg_pretend`.

Reply:
I've been not comfortable with this solution either. I'll look into a better solution, hang on.

I wonder if it should be checked at all. It's no hard requirement of the daemon (it will just print a log saying that `SCHED_ISO` is not available and continue), and people who install ck-sources or pf-sources mostly do it because of this scheduler. I think, ck even defaults to `SCHED_MUQSS`.

What do you think: Remove or revise?

Reply:
I think the first question to ask is: what is thing exactly, and what do I [the user] gain by enabling it? If it's real gain and you can explain it to the user, I think it's worth checking in detail. If it's more effort than gain, then meh.

Reply:
I've revised the checks and the explanation. It is a bit lengthy now, hope that's okay.